### PR TITLE
article tags added

### PR DESCRIPTION
**Pull request by luc-tuyishime**
#### What does this PR do?
- Allow users  to add tags on articles they have created

#### Description of Task to be completed?
- Create tags and display on articles views
#### How should this be manually tested?
 - After cloning the repo, cd into it and type npm test.
 - After cloning the repo, cd into it and type npm start and follow the instruction to open it in the browser.

#### Any background context you want to provide?
- To be able to  tag article you have to be authenticated
#### What are the relevant pivotal tracker stories?
[#165412882](https://www.pivotaltracker.com/story/show/165412882)
#### Screenshots (if appropriate)
image

### CODE REVIEW

File: src/components/Articles/Article/Article.js [hunk 1]
```diff
@@ -207,6 +207,16 @@ export class Article extends Component {
...
                 </div>
                 <div className="divider light" />
+                <div>
+                  {article.tagList
+                    ? article.tagList.map(value => (
+                        <p className="tags-values" key={value}>
+                          <span className="value-tag">{value}</span>
+                        </p>
+                    ))
+                    : 'No tags added'}
```
Comment (kagaramag) [inline src/components/Articles/Article/Article.js]: If there is no tag added to the article, I don't think it is not needed to tell visitors that there is no `tag added`.

File: src/store/initialStates/tagsInitialState.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  addTags: {
+    loading: false,
+    response: '',
+    errors: {}
+  },
+  getTags: {
+    loading: false,
+    message: '',
+    errors: {}
+  }
```
Comment (kagaramag) [inline src/store/initialStates/tagsInitialState.js:6]: I think getting tags is already an article object. No need to fetch them again

File: src/components/Articles/Article/Article.js (hunk 1 above)
Comment (luc-tuyishime) [reply src/components/Articles/Article/Article.js]: Yes, you are right, I have already removed the message.

File: src/store/initialStates/tagsInitialState.js (hunk 2 above)
Comment (luc-tuyishime) [reply src/store/initialStates/tagsInitialState.js:6]: Yes yes already removed

**Review by kagaramag — APPROVED** [review]

**Pull request merged**